### DISCUSSION TITLE
Delete branch after merge (not automatically)

### DISCUSSION THREAD

**Discussion by GrimalDev** [pwntester/octo.nvim#572] [Ideas]
Hi,

How about about deleting the merged branch after merge. Not automatically, just a neat option to select to delete current branch. 

Else, i'm thinking of making a shortcut that makes 3 git commands.
- Checkout to base branch
- Delete old local head brach
- Delete old origin head branch.

Of course this would suffice, but I am thinking that a direct integrated option would be cool.

What do you think?

**Comment by williambdean** [comment]
I think this can already be done. I think it is part of `Octo pr merge`'s additional flags?

**Reply by GrimalDev** [reply]
> I think this can already be done. I think it is part of `Octo pr merge`'s additional flags?

Yes, thank you

```
Octo pr merge <delete / nodelete>
```
And there has been a default parameter added in #658 
```
  default_delete_branch = false,           -- whether to delete branch when merging pull request with either `Octo pr merge` or from picker (can be overridden with `delete`/`nodelete` argument to `Octo pr merge`)
```

**Reply by williambdean** [reply]
Nice. Good to hear. I will resolve the discussion in the meantime. Feel free to add a new issues / discussion if needed
